pkg/baseboard/baseboard_windows.go

```diff
@@ -6,7 +6,7 @@
 package baseboard
 
 import (
-	"github.com/StackExchange/wmi"
+	"github.com/yusufpapurcu/wmi"
```

Comment:
yes, the README on "github.com/StackExchange/wmi" recommends this fork, so it seems fine